### PR TITLE
Fixed for Swift 3

### PR DESCRIPTION
**Pull request by rafaelc0sta**
Minor fixes for swift 3. Fully functional. However, the method naming is not entirely adequate. Many methods should be refactored.

### CODE REVIEW

**Comment by philackm** [comment]
Thank you for doing this!

This is definitely a priority, I want to have it working with Swift 3 as well. I'm busy with uni assignments at the moment, I will check this and try to deal with the method naming over the next week.

**Comment by philackm** [comment]
I might even just double check that everything works and merge this then deal with the major issues over the summer.

**Comment by rafaelc0sta** [comment]
@philackm I did a lot of it in this second commit of mine. Check it out, I think it is good to go.

Only one change I'd like to comment: -prepareForInterfaceBuilder
I commented -setData: on it because it was crashing my Interface Builder. Could be something on my machine, but worth checking out.

**Comment by iceman201** [comment]
omg my project is waiting for this framework support swift 3

**Comment by devJoshLopez** [comment]
please merge this. Much love

File: Classes/ScrollableGraphView.swift [hunk 1]
```diff
@@ -3,20 +3,20 @@ import UIKit
 // MARK: - ScrollableGraphView
 @IBDesignable
 @objc public class ScrollableGraphView: UIScrollView, UIScrollViewDelegate, ScrollableGraphViewDrawingDelegate {
-    
+
```
Comment (mikeMTOL) [inline Classes/ScrollableGraphView.swift]: This is sloppy.. you shouldn't commit empty line changes.

File: Classes/ScrollableGraphView.swift [hunk 2]
```diff
@@ -667,167 +666,168 @@ import UIKit
...
-    }
-    
-    private func calculateRangeForActivePointsInterval(interval: Range<Int>) -> (min: Double, max: Double) {
-        
+
+        let actualMin = clamp(value: min - numberOfPointsOffscreen, min: minPossible, max: maxPossible)
+        let actualMax = clamp(value: max + numberOfPointsOffscreen, min: minPossible, max: maxPossible)
+
+
+        return actualMin ..< actualMax
```
Comment (mikeMTOL) [inline Classes/ScrollableGraphView.swift]: this should me `actualMin ..< actualMax + 1` to conform to what it was before.

File: Classes/ScrollableGraphView.swift [hunk 3]
```diff
@@ -1159,88 +1163,88 @@ private class LabelPool {
...
             return location.x
         }
         set {
             location.x = newValue
         }
     }
-    
-    private var y: CGFloat {
+
+    var y: CGFloat {
```
Comment (mikeMTOL) [inline Classes/ScrollableGraphView.swift]: this should stay `fileprivate` to maintain previous behaviour

**Comment by dhf** [comment]
DataPoints are not drawn correctly if we have many of them. To test these behavior start sample project in original repo!

**Comment by philackm** [comment]
I have decided to merge in #48 to perform the Swift 3 migration. Thanks everyone.

**Pull request closed without merging**